tests/tries_test.go

```diff
@@ -0,0 +1,76 @@
+package tests
```

Comment:
Here we have the benchmarks from https://github.com/gballet/go-ethereum/pull/146.
I only removed the statelessness one as we discussed, so we avoid test flags.